src/foam/dao/CSVSink.js

```diff
@@ -40,22 +70,43 @@ foam.CLASS({
       name: 'isNewLine',
       value: true,
       visibility: 'HIDDEN'
+    },
+    {
+      name: 'axioms',
+      factory: function() {
+        return this.of.getAxiomByName('tableColumns').columns;
```

Comment:
I think we should name these columns and columns_ instead of axioms and props, to be consistent with TableView.